get-pmm.sh

```diff
@@ -223,19 +223,25 @@ run_docker() {
 # If any PMM server instance is run - stop and backup it.
 #######################################
 start_pmm() {
-  msg "Starting PMM server..."
+  msg "Fetching docker image...\n"
   run_docker "pull $repo:$tag 1> /dev/null"
 
+  msg "Checking data volume...\n"
+  pmm_archive="$(date "+%F-%H%M%S")"
   if ! run_docker "inspect pmm-data 1> /dev/null 2> /dev/null"; then
     run_docker "create -v /srv/ --name pmm-data $repo:$tag /bin/true 1> /dev/null"
-    msg "Created PMM Data Volume: pmm-data"
+    msg "Created PMM Data Volume: pmm-data\n"
+  else 
+    msg "\n\t${YELLOW}Existing PMM Data volume found, renaming to pmm-data-$pmm_archive${NOFORMAT}\n"
+    run_docker "rename pmm-data pmm-data-$pmm_archive"
+    run_docker "create -v /srv/ --name pmm-data $repo:$tag /bin/true 1> /dev/null"
+    msg "Created PMM Data Volume: pmm-data\n"
```

Comment:
You can move these two strings out of the `if` block.

Reply:
@BupycHuk made a good point that people use this for upgrades (as evidenced by the ticket @denisok shared) so I'm rethinking this.  I think as a best practice we *should* be at least taking a backup if we find an existing pmm-data volume and perhaps allow users to skip it if they run in interactive mode.  Thoughts?